components/script/dom/webidls/BrowserElement.webidl

```diff
@@ -83,6 +83,21 @@ dictionary BrowserShowModalPromptEventDetail {
   // TODO(simartin) unblock() callback
 };
 
+dictionary BrowserElementOpenTabEventDetail {
+  // https://developer.mozilla.org/en-US/docs/Web/Events/mozbrowseropentab
+  DOMString url;
+  // TODO(ajeffrey) this isn't part of the Browser API.
+  DOMString target;
```

Comment:
for reference: http://mxr.mozilla.org/mozilla-central/source/dom/browser-element/BrowserElementChildPreload.js#703